automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -30,9 +30,9 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
           cluster.serviceAccountInfo.notebookServiceAccount shouldBe None
 
           withWebDriver { implicit driver =>
-            withNewNotebook(cluster) { notebookPage =>
+            withNewNotebook(cluster, PySpark2) { notebookPage =>
               // should not have notebook credentials because Leo is not configured to use a notebook service account
-              verifyNoNotebookCredentials(notebookPage)
+              verifyNoNotebookCredentials(notebookPage, PySpark2, petEmail)
```

Comment:
The name `verifyNoNotebookCredentials` sounds misleading to me. Given what it does, shouldn't it be something like `verifyNotebookCredentials`, or something more explanatory?

Reply:
But I now see that there _is_ also a `verifyNotebookCredentials`, which verifies a subset of things. My suggestion would be to refactor those two methods like so:

```scala
def verifyOauth2Client()
def verifyFiss()
def verifySpark()
def verifyMetadataServer() // preferably something even more specific like verifyScopes()
```
and then if it is necessary, we can have some convenience methods that bundle some of those together as necessary.

Reply:
👍 , agreed that looks much better.